docs/rooosss/style/footer.css

```diff
@@ -0,0 +1,58 @@
+.footer__inner {
+  padding: 30px 0 40px;
+}
+
+.footer__container {
+  background-color: #EF4934;
+}
+
+.footer__image-container {
+  margin-bottom: 32px;
+  display: flex;
+  justify-content: center;
+  gap: 30px;
+}
+
+footer__image-container a {
```

Comment:
this selector is broken and useless. The `a` elements have `cursor: pointer;` by default
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/ffb38f69-b069-4f0b-b056-913b7530c62a)